### PR TITLE
Dev monitor microsoft.insights 2020 05 01 preview

### PR DESCRIPTION
**Pull request by moshemal**
Please refer to the private repository PR:
https://github.com/Azure/azure-rest-api-specs-pr/pull/1451

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/monitor/resource-manager/readme.md [hunk 1]
```diff
@@ -28,14 +28,23 @@ These are the global settings for the MonitorClient API.
 title: MonitorClient
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2019-06
+tag: package-preview-2020-05
```
Comment (MyronFanQiu) [inline specification/monitor/resource-manager/readme.md:31]: I don't think we should change this one.

File: specification/monitor/resource-manager/readme.md [hunk 2]
```diff
@@ -44,7 +53,7 @@ input-file:
 
 These settings apply only when `--tag=package-2020-03` is specified on the command line.
 
-```yaml $(tag) == 'package-2020-03'
+``` yaml $(tag) == 'package-2020-03'
```
Comment (MyronFanQiu) [inline specification/monitor/resource-manager/readme.md:56]: Could you remove these changes? Looks like it's a regression.

File: specification/monitor/resource-manager/readme.md (hunk 1 above)
Comment (moshemal) [reply specification/monitor/resource-manager/readme.md:31]: @MyronFanQiu  
I used Microsoft [OpenApiHub](https://portal.azure-devex-tools.com) and it committed 27d3d19af, 847a13e8b, 2a9e0ce87  which include auto generated changes to the readme file. I fixed this line and all other by using the version from the private repository (which I changed manually). Please review.

File: specification/monitor/resource-manager/readme.md (hunk 2 above)
Comment (moshemal) [reply specification/monitor/resource-manager/readme.md:56]: same as [above](https://github.com/Azure/azure-rest-api-specs/pull/10101#discussion_r453459093)

**Review by mmyyrroonn — APPROVED** [review]
Approved since it's already reviewed in private repo. Waiting for ARM's confirmation

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed.

**Comment by moshemal** [comment]
@chiragg4u 
Anything else to do to make this PR approved?

**Review by pilor — APPROVED** [review]
Port of already signed off PR in private repo

**Comment by moshemal** [comment]
@MyronFanQiu ping

**Comment by moshemal** [comment]
@MyronFanQiu

**Pull request merged**